.github/workflows/publish-claudio-review.yml

```
@@ -1,25 +1,24 @@
-      reviewed_head_sha: {description: "Head SHA analyzed by the reviewer", required: true, type: string}
-      event: {description: Review event, required: true, default: COMMENT, type: choice, options: [COMMENT]}
-      review_body: {description: "Review summary in Markdown", required: false, type: string}
-      inline_comments_json: {description: "JSON array of path, line, and body findings", required: false, default: '[]', type: string}
-      replies_json: {description: "JSON array of comment_id and body replies", required: false, default: '[]', type: string}
-      resolve_thread_ids_json: {description: "JSON array of thread node IDs", required: false, default: '[]', type: string}
-permissions: {contents: read}
+      pr_number: { description: Pull request number, required: true, type: string }
+      event: { description: Review event, required: true, default: COMMENT, type: choice, options: [COMMENT] }
+      review_body: { description: Review summary in Markdown, required: false, default: '', type: string }
+      reviewed_head_sha: { description: Full PR head SHA reviewed, required: true, type: string }
+      inline_comments_json: { description: JSON array of path, line, and body findings, required: false, default: '[]', type: string }
```

actionlint (1.7.12)

[error] unexpected key "line" for "inputs" section. expected one of "default", "description", "required"

(syntax-check)